src/api/app/helpers/application_helper.rb

```diff
@@ -1,6 +1,8 @@
 # Methods added to this helper will be available to all templates in the application.
 module ApplicationHelper
-  def render_diff(content)
-    sanitize(CodeRay.scan(content, :diff).div(line_numbers: :inline, css: :class))
+  def render_diff(content, filename:)
+    filename.gsub!(/[^\w-]/, '_')
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id mentions the values format

Reply:
I switched to use the file index, instead of the file name.